Dispatcher/Supporting Files/AppDelegate.swift

```diff
@@ -14,6 +14,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let response = readApiKeyFromConfigFile()
```

Comment:
Instead of "response" a better naming would be what it actually is - apiKeyValue